packages/client/src/async-completion-client.ts

```diff
@@ -216,23 +216,23 @@ export class AsyncCompletionClient extends BaseClient {
     const payloads = await encodeToPayloads(this.dataConverter, details);
     let cancelRequested = false;
     try {
-      let response: temporal.api.workflowservice.v1.RecordActivityTaskHeartbeatResponse;
       if (taskTokenOrFullActivityId instanceof Uint8Array) {
-        response = await this.workflowService.recordActivityTaskHeartbeat({
+        const response = await this.workflowService.recordActivityTaskHeartbeat({
```

Comment:
If anyone wonders, the `Response` interface for both of these calls was previously identical, so assigning both to a single `response` var was ok. But a new prop has been added to one of them.